packages/lb-components/src/components/pointCloudView/hooks/useStatus.ts

```diff
@@ -50,24 +57,45 @@ export const useStatus = () => {
 
     switch (toolName) {
       case EToolName.Rect:
+        sideViewInstance?.toolInstance.clearResult()
+        backViewInstance?.toolInstance.clearResult()
+
         topViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
         topViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
-        // polygon2dOperation.setPattern(EPolygonPattern.Rect);
+        sideViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
+        sideViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
+        backViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
+        backViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
```

Comment:
why not use `forEach`?

[topViewInstance, backViewInstance,sideViewInstance].forEach(() => xxx);

Reply:
Nice try